Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPage.mm

```diff
@@ -883,6 +886,40 @@ static void swizzledPerformTextSearchWithQueryString(id, SEL, NSString *, UIText
     testPerformTextSearchWithQueryStringInWebView(webView.get(), @"Birthday", searchOptions.get(), 0UL);
```

Comment:
We should add `hasPerformedTextSearchWithQueryString = false;` here to leave things in the right state.